app/decorators/order_detail_decorator.rb

```diff
@@ -0,0 +1,6 @@
+class OrderDetailDecorator < SimpleDelegator
+
+  delegate :full_name, to: :assigned_user, prefix: :assigned_user, allow_nil: true
```

Comment:
`User#to_s` is defined as `full_name`, so you might be able to get away with the same thing here where we just say `od.assigned_user` and it gives us the full name or `""` if it's nil.